installer_scripts/install-yb-voyager

```diff
@@ -389,7 +389,13 @@ create_base_ora2pg_conf_file() {
 		output "Overwriting ${conf_file_name}. Existing contents are backed up in ${backup_file_name}"
 	fi
 	output "Installing the latest base-ora2pg.conf"
-	sudo wget -nv -O $conf_file_name https://github.com/yugabyte/yb-voyager/raw/main/yb-voyager/src/srcdb/data/sample-ora2pg.conf
+
+	if [ "${VERSION}" == "latest" ]
+	then
+		sudo wget -nv -O $conf_file_name https://github.com/yugabyte/yb-voyager/raw/$YB_VOYAGER_GIT_HASH/yb-voyager/src/srcdb/data/sample-ora2pg.conf
+	else
+		sudo cp ../yb-voyager/src/srcdb/data/sample-ora2pg.conf $conf_file_name
```

Comment:
You should not assume that the script is being run from the `installer_scripts/` directory.

There is already an example where we first find the root of the git repo and switch to it:
```
        pushd `git rev-parse --show-toplevel` > /dev/null
        pushd yb-voyager > /dev/null
        $GO install
        popd > /dev/null
        popd > /dev/null
```
While using the above example for reference, you won't have to `cd` into the `yb-voyager` directory.